### ISSUE TITLE
Salesforce Inspector is not working in Chrome Browser

### ISSUE THREAD

**Issue by Praveen663** [sorenkrabbe/Chrome-Salesforce-inspector#176]
Hi Team,

When we open the salesforce records in salesforce inspector, there it is showing as "No records to display".

I tried uninstalling and then installing again, still issue persist.

Regards,
Praveen

![Salesforce Inspector issue](https://user-images.githubusercontent.com/72329673/95007066-deb82280-0628-11eb-90a3-40751cc78e67.PNG)

**Comment by enkodellc** [comment]
This is correct functionality as you have not searched for anything.. therefore no records to display. Type in Case or Account in the search box and you will see records / results

**Comment by Praveen663** [comment]
Earlier, whenever we click on inspector it used to display the "show data" option. Now, that option is not available.

FYI: I have open the record, then click on inspector. it means, all the account details should be displayed in the inspector.

Inside of Inspector, we can search which field we want to refer or search.

Regards,
Praveen

**Comment by dufoli** [comment]
hello,
this addon is not maintain anymore. I work on a forked project.
The current record and data is displayed now and you can used show all data button to get detail.
You should try it.
https://chromewebstore.google.com/detail/salesforce-inspector-adva/dbfimaflmomgldabcphgolbeoamjogji
source:
https://github.com/dufoli/Salesforce-Inspector-Advanced
